debug_toolbar/static/debug_toolbar/js/toolbar.js

```diff
@@ -8,6 +14,10 @@ function onKeyDown(event) {
 
 const djdt = {
     handleDragged: false,
+    abort() {
+        controller.abort();
+        resetAbortController();
+    },
```

Comment:
This looks like the only part of the code that uses the controller and the signal. Outside of allowing another third party panel to abort a request, what purpose does this serve?

Reply:
Calling abort will unregister the click handlers - the intent here is that init will configure the JS click handlers and this will undo it to put us back in the state we were immediately before init.

  The Purpose of that is that for boosted requests / Turbolinks the new page will contain it's own toolbar html, and I was finding conflicts with multiple click handlers registered.